general-javascript-best-practices.md

```diff
@@ -149,6 +149,18 @@ Use instead of underscore, due to correct semantic versioning, better performanc
 
 Please only require the methods you need rather than the whole library in order to keep build sizes at a minimum.
 
+**Good:**
```

Comment:
I think we should prefer that people don't use `_.chain` in clientside code. There is no way of using that with this approach. We also lose all the benefits of doing it this way if someone requires the whole of lodash just to use `_.chain`. Thoughts?